model/build.gradle.kts

```diff
@@ -51,7 +51,7 @@ smallryeOpenApi {
   schemaFilename.set("META-INF/openapi/openapi")
   operationIdStrategy.set("METHOD")
   scanPackages.set(
-    listOf("org.projectnessie.api", "org.projectnessie.api.http", "org.projectnessie.model")
+    listOf("org.projectnessie.api.v2", "org.projectnessie.api.v2.http", "org.projectnessie.model")
```

Comment:
Q: Don't we need v1 here as well?

Reply:
maybe publish v1 + v2beta now and remove v1 later

Reply:
yes, makes sense... will do.

This will probably touch the "main" commit with some changes in API classes to re-review... just FYI :)

On the positive side, this should help to clean-up the Swagger page in Quarkus dev UI.

Reply:
switched to `v2beta` URLs

Reply:
Using `v2-beta` OpenAPI tags instead now.